pkg/config/config.go

```diff
@@ -213,6 +213,7 @@ func init() {
 	BindEnvAndSetDefault("logs_config.dev_mode_use_proto", false)
 	BindEnvAndSetDefault("logs_config.run_path", defaultRunPath)
 	BindEnvAndSetDefault("logs_config.open_files_limit", 100)
+	BindEnvAndSetDefault("logs_config.collect_all", false)
```

Comment:
`collect_all` seems very vague, @NBParis how would you name this? `collect_all_docker_logs`?

Reply:
@tmichelet Indeed it is clearer but it is so long.
What do you think about `logs_config.docker_collect_all` ?

Reply:
ok

Reply:
I'd vote for `logs_config.container_collect_all`